shared/common-adapters/button.desktop.js

```diff
@@ -144,66 +144,65 @@ const buttonCommon = {
   minWidth: 10
 }
 
-const styles = {
-  buttonPrimary: {
-    ...buttonCommon,
-    backgroundColor: globalColors.blue,
-    disabledOpacity: 0.2,
-    marginRight: 10
-  },
-  buttonSecondary: {
-    ...buttonCommon,
-    backgroundColor: globalColors.lightGrey2,
-    disabledOpacity: 0.3,
-    marginRight: 10
-  },
-  buttonDanger: {
-    ...buttonCommon,
-    backgroundColor: globalColors.red,
-    disabledOpacity: 0.2,
-    marginRight: 10
-  },
-  buttonFollow: {
-    ...buttonCommon,
-    backgroundColor: globalColors.green,
-    disabledOpacity: 0.3,
-    marginRight: 10,
-    minWidth: 125
-  },
-  buttonFollowing: {
-    ...buttonCommon,
-    backgroundColor: globalColors.white,
-    border: `solid 2px ${globalColors.green}`,
-    marginRight: 10,
-    minWidth: 125
-  },
-  buttonUnfollow: {
-    ...buttonCommon,
-    backgroundColor: globalColors.blue,
-    disabledOpacity: 0.2,
-    marginRight: 10,
-    minWidth: 125
-  },
-  buttonSmall: {
-    height: 28,
-    lineHeight: '24px'
-  },
-  buttonLabel: {
-    paddingLeft: 25,
-    paddingRight: 25
-  },
-  buttonSmallLabel: {
-    ...globalStyles.fontRegular,
-    paddingLeft: 20,
-    paddingRight: 20
-  },
-  progress: {
-    position: 'absolute',
-    height: 'calc(100% - 4px)',
-    left: 0,
-    right: 0,
-    top: 0,
-    bottom: 0,
-    margin: 'auto'
-  }
+const stylesButtonPrimary = {
+  ...buttonCommon,
+  backgroundColor: globalColors.blue,
+  disabledOpacity: 0.2
```

Comment:
marginRight: 10 removed here. i think this just causes problems, our center aligned buttons are slighly off, the stretched ones also. in practice there's nothing 'to the right' of the button so we don't really want this